tensorflow/core/kernels/queue_ops.cc

```diff
@@ -425,6 +425,33 @@ class QueueSizeOp : public QueueOpKernel {
 REGISTER_KERNEL_BUILDER(Name("QueueSize").Device(DEVICE_CPU), QueueSizeOp);
 REGISTER_KERNEL_BUILDER(Name("QueueSizeV2").Device(DEVICE_CPU), QueueSizeOp);
 
+// Defines a QueueIsClosedOp, which returns true if a given Queue is closed
+// and false if it is open.
+//
+// The op has one input, which is the handle of the appropriate Queue;
+// and one output, which is a scalar boolean value that shows whether
+// the queue is closed.
```

Comment:
Remove this comment: the REGISTER_OP documentation should cover this.  This kind of comment is in danger of going out of date.